Documentation/Gifbuilder/ObjectNames/Index.rst

```diff
@@ -197,8 +184,8 @@ different each time!
          This will allow for the closest color to be matched instead. You may
          need this if you image is not guaranteed "clean".
 
-         **Note:** You may experience that this does not work, if you use the
-         reduceColors-option or render text with niceText-option.
+         **Note:** You may experience that this does not work, if you render
+         text with the niceText option.
```

Comment:
```suggestion
         text with the :typoscript:`niceText` option.
```
I think this is a TS option?

Reply:
I assume so, I would like to link that (if it is an option, but this would be another PR). But the Gifbuilder docs need more love anyway.

Reply:
Ok, lets do it in a follow up then